core/controllers/reader.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Controllers for the Oppia exploration learner view."""
+"""Controllers for the Oppia exploration user view."""
```

Comment:
Why the change here? We distinguish between our users who create explorations ("creators") and users viewing explorations ("learners"). This is consistent throughout the codebase.